src/sentry/api/endpoints/organization_details.py

```diff
@@ -116,6 +123,48 @@ def update_organization_scenario(runner):
         )
 
 
+class TrustedRelaySerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        return convert_dict_key_case(instance, snake_to_camel_case)
+
+    def to_internal_value(self, data):
+
+        key_name = "{unknown}"
```

Comment:
This is dead code, since you set `key_name` two lines below. Please also remove the empty line above.

Reply:
yep, disappeared after refactor, (`key_name="{unknown}"` was used for the case when `key_name=data.get('name')` raised (i.e. we didn't get a dictionary in the call) and I didn't get to initialise it before I generated the error message ).